bin/Bin/FormatOptions.purs

```diff
@@ -65,4 +82,56 @@ unicodeOption =
         "Unicode punctuation is never preferred."
         $> UnicodeNever
     ]
-    # Arg.default UnicodeSource
+    # Arg.default defaults.unicode
+
+fromJson :: Json -> Either JsonDecodeError FormatOptions
```

Comment:
Should this and the defaults both move into `Bin.FormatOptions`? We don't use the JSON in the tests, but we do use defaults, and the `fromJson` would make more immediate sense if imported from that module.